File: 3462-vowels-game-in-a-string/vowels-game-in-a-string.cpp
class Solution {
public:
    bool doesAliceWin(string s) {
        unordered_map<char,int> mp;
        vector<char> v ={'a','e','i','o','u'};
        for(auto i:v) mp[i]++;
        int vc = 0;
        for(auto i : s){
            if(mp[i]) vc++;
        }
        if(vc==0) return false;
        return true;


    }
};

File: 3462-vowels-game-in-a-string/vowels-game-in-a-string.java
class Solution {
    public boolean doesAliceWin(String s) {
        int vowelsCount =0 ;
        HashMap<Character, Integer> hashMap = new HashMap<>();
        char[] vowels = {'a','e','i','o','u'};
        for(int i =0 ; i < 5 ; i++){
            hashMap.put(vowels[i],1);
        }
        for(int i =0 ; i < s.length();i++){
            if(hashMap.containsKey(s.charAt(i))) vowelsCount++;
        }
        return (vowelsCount  > 0) ;
    }
}